### ISSUE TITLE
Rigatoni

### ISSUE THREAD

**Issue by ianjhoffman** [VCVRack/community#775]
Just finished my first VCV Rack module - it's a small waveshaper/utility module that emulates TZFM, based on this [ModWiggler topic](https://www.modwiggler.com/forum/viewtopic.php?t=277734).

GitHub repo: https://github.com/ianjhoffman/RigatoniModular

**Comment by AndrewBelt** [comment]
Your plugin slug is named after 1 module in the plugin. What if you add more modules to the plugin? Would it make more sense to name the plugin "Rigatoni" with the slug `Rigatoni`?

**Comment by ianjhoffman** [comment]
@AndrewBelt is it generally best to have all modules from a virtual "manufacturer" in one repo<->plugin?

**Comment by ianjhoffman** [comment]
Updated the issue title, repo name, and manifest to better reflect your suggestion!

**Comment by AndrewBelt** [comment]
Usually, yes. If you had 20 plugins instead of 20 modules in 1 plugin, it'd be a nightmare to push/build/test/update them all.
Thanks, the next step is to wait on our review and build!

**Comment by cschol** [comment]
Plugin fails to load: https://github.com/ianjhoffman/RigatoniModular/issues/1

**Comment by ianjhoffman** [comment]
Should be working now!

**Comment by cschol** [comment]
New issue logged in https://github.com/ianjhoffman/RigatoniModular/issues/1. Panel fails to load.

**Comment by ianjhoffman** [comment]
Thanks for keeping up with these - that should be resolved too :)

**Comment by cschol** [comment]
Your plugin has been integrated and will be available for all supported platforms with the next Plugin Manager update. Please let us know in **this** issue when a new version is available. We will reopen it. Thank you for providing your plugin to the Rack community.

**Comment by ianjhoffman** [comment]
Hi! New version of `Rigatoni` is available with an additive voice, **Loom**. V2.2.0

**Comment by cschol** [comment]
Compile failure on macOS platform: https://github.com/ianjhoffman/RigatoniModular/issues/6

**Comment by ianjhoffman** [comment]
Confirmed recent changes cause the plugin to build on all 4 target platforms using https://github.com/VCVRack/rack-plugin-toolchain
